build/packages-template/bbb-config/after-install.sh

```diff
@@ -130,7 +130,7 @@ if [ -d /var/mediasoup/screenshare ]; then
 fi
 
 sed -i 's/worker_connections 768/worker_connections 4000/g' /etc/nginx/nginx.conf
-echo 'limit_conn_zone $uri zone=ws_zone:5m;' > /etc/nginx/conf.d/html5-conn-limit.conf
+
```

Comment:
Add something like `if [ -f /etc/nginx/conf.d/html5-conn-limit.conf ]; then rm /etc/nginx/conf.d/html5-conn-limit.conf fi`
because your new file name does not match the old file name. The new name is better, so let's just ensure cleaning up on upgrade instead of changing the new name

Reply:
Good catch, thank you! I added it to `bbb-html5/after-install.sh`, sounds good?